README.md

```diff
@@ -115,5 +115,11 @@ only the `build` directory is deployed, so you should `npm build` before deployi
 ## CI/CD
 CircleCI will test all commits pushed to the repo. As soon as a branch is merged
 to the `master` branch, it will (assuming tests pass) additionally deploy to the
-`smart-cities-dev` instance. To deploy to the `smart-cities-staging` instance,
-simply push to the `staging` branch.
+`smart-cities-staging` instance.
+
+Production deploys are triggered when a tag is pushed with a name in the form `release-*`.
+To deploy, simply: 
+
+```
+git tag `date "+release-%Y%m%d%H%M%S"` && git push --tags
```

Comment:
Is it worth aligning how we do tagging with other web projects? Using `v1.0.0` format is common (e.g. [npm releases](https://github.com/npm/npm/releases)) and [GitHub recommends](https://help.github.com/articles/creating-releases/) using semver for tags.

There's an npm command for this: `npm version`. That bumps the version in `package.json` and creates a git tag too. https://docs.npmjs.com/cli/version

So the instructions would be: run `npm version major` and that bumps the version and commits it with the appropriate tag.

Thoughts?

Reply:
Good idea. The format was copied from performance dashboard; since this is a purely JavaScript app, using npm would work. 

I notice that `npm version major` (or `minor`, etc) commits but doesn't push.

I'll also need to change the tag filter config to pick up that format. `/^v\d+\.\d+\.\d+/` should work, I think (or even just `/^v\d/`)

Reply:
You could add an npm script `"deploy": "npm version major && git push --tags"` I think would work.

No point in doing anything other than major since it isn't code that has an API.